docs/devenv_cloud.md

```diff
@@ -0,0 +1,142 @@
+* The RHEL for Edge virtual machine running as a **nested** virtual instance on the development host
+* The Ansible server virtual machine running as a **nested**  virtual instance on the development host
+
+This configuration provides for the required functionality and flexibility neccessary for the MicroShift development and testing environments.
+
+### AWS Cloud
+
+Start by selecting the right region providing the necessary hardware for effective support of nested virtualization.
+* `x86_64` environment requires [c6i instances](https://aws.amazon.com/ec2/instance-types/c6i/)
+* `aarch64` environment requires [c7g instances](https://aws.amazon.com/ec2/instance-types/c7g/) 
+> The Europe (Ireland) `eu-west-1` or US East (N. Virginia) `us-east-1` regions are recommended.
+
+Review and configure the following VPC resources.
+* A default VPC settings should be sufficient
+* A default subnet in one of the availability zones with access to the Internet
+
+Review and configure the following EC2 resources.
+* A default security group with `Inbound rules` set to **only** allow SSH traffic on port 22. It is recommended to further restrict access to the SSH port from your IP address.
+* Import your `Key Pair` used for accessing the virtual instances
+
+Proceed by creating a virtual instance using the following parameters.
+* Use a descriptive name prefixed by your user name if the account is shared with others (i.e. `myuser-ushift-dev`)
+* Select the Red Hat Linux operating system family
+* Select the `64-bit (x86)` or `64 bit (Arm)` architecture
+* Select Red Hat Enterprise Linux 8 (HVM) operating system. 
+> Do not attempt to use RHEL 9 as it is not supported for the development environment. The operating system will be upgraded to the 8.7 version in the configuration stage.
+* Select the instance type depending on your architecture of choice:
+    * `c6i.2xlarge` for `x86_64`
+    * `c7g.2xlarge` for `aarch64`
+* Select your `Key pair`
+* Select your security group, **only allowing** SSH port 22 inbound traffic
+* Configure storage with the following volumes:
+    * 100 GiB root volume on `gp3` storage type
+    * 10 GiB data volume on `gp3` storage type
+
+Review your selections and lauch the virtual instance. 
+> Unless noted otherwise, all the further configuration should be performed on the cloud virtual instance created in this section.
+
+### Other Cloud Environments
+
+Other public cloud vendors and environments should also be suitable for running the MicroShift development environment, provided the appropriate hardware and operating system configuration is available.
+
+## Development Virtual Machine
+
+Log into the virtual instance running in the cloud using the appropriate login credential (i.e. `ec2-user` for AWS) and a private SSH key.
+
+```bash
+ssh -i ~/.ssh/<private_key> ec2-user@<instance_public_IP>
+```
+
+Create and configure the `microshift` user account, also allowing SSH logins using a password.
```

Comment:
Do we need to insist on creating `microshift` user? Can't we proceed with ec2-user instead? Do we have it hard coded somewhere? Or is it just consistency with amd64 devenvs?

Reply:
Yes, some scripts depend on microshift user. It's good to have consistency and working from ec2-user account is not recommended anyway.